opm/simulators/linalg/bda/opencl/openclBILU0.cpp

```diff
@@ -189,6 +195,14 @@ create_preconditioner(BlockedMatrix<Scalar>* mat, BlockedMatrix<Scalar>* jacMat)
 
     auto *matToDecompose = jacMat ? jacMat : mat;
 
+    if (jacMat) {
+#if HAVE_OPENMP
+        if (omp_get_max_threads() > 1) {
+           copyThread->join();
+        }
+#endif
```

Comment:
So without OpenMP we simply use no threads. If I configure OPM on a multicore machine without threads then copying will be done sequentially. Why do we want that?